queries/reports/groups.js

```diff
@@ -44,8 +15,9 @@ WHERE {
           dossier:behandelt ?case.
         ?case
           a dossier:Dossier .
+        ?case dossier:Dossier.isNeerslagVan ?decisionmakingFlow .
```

Comment:
Same here. Remove the links to `case` and `decisionmakingFlow` and use `OPTIONAL { ?publicationFlow pub:beleidsveld ?policyDomain . }`